controllers/plans_controller.js

```diff
@@ -35,5 +35,17 @@ exports.getPlanByBatchId = async (req, resp) => {
     }
 }
 
+exports.getAllPlans = async (req, resp) => {
+    try {
+        var input_response = await plan_manager.pre_process_get_all_plans(req)
+        var process_response = await plan_manager.process_get_all_plans_input_req(input_response)
+        var post_process_response = await plan_manager.post_process_get_all_plans(process_response, resp)
+    } catch (e) {
+        const status_code = e.statusCode ? e.statusCode : 500
+        return resp.status(status_code).send({ status: "Failure", message: e.name })
```

Comment:
pass "Internal Server Error" in message .Also console.log(e) before sending response in catch.